typescript-client/development/changelog.mdx

```diff
@@ -2,6 +2,20 @@
 title: Changelog
 ---
 
+# `0.0.510`
+
+## Breaking Changes
+
+Revamp of the OpenAI instrumentation : instead of manually instrumenting each call result, you can now instrument OpenAI globally. Each subsequently call will be logged with no additional code.
+
+This new implementation is context-aware and will log the OpenAI calls in the right steps and threads if you use the new wrapper syntax.
```

Comment:
I like when repos document the error I could be getting. 

If I upgraded to `0.0.510` without changing my instrumentation I could read here that my error message is expected and have an example of the new wrapper syntax.

Reply:
good point, i added a block detailing that